phpunit.xml.dist

```diff
@@ -1,12 +1,13 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<phpunit bootstrap="vendor/autoload.php"
-         colors="true"
-         convertErrorsToExceptions="true"
-         convertNoticesToExceptions="true"
-         convertWarningsToExceptions="true">
-    <testsuites>
-        <testsuite name="Europa Search client Test suite">
-            <directory>./tests</directory>
-        </testsuite>
-    </testsuites>
+<phpunit xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" bootstrap="vendor/autoload.php" colors="true" convertErrorsToExceptions="true" convertNoticesToExceptions="true" convertWarningsToExceptions="true" xsi:noNamespaceSchemaLocation="https://schema.phpunit.de/9.3/phpunit.xsd">
```

Comment:
remove convertErrorsToExceptions default is true
remove convertNoticesToExceptions default is true
remove convertWarningsToExceptions default is true
xsi:noNamespaceSchemaLocation is schema for phpunit 9.3, latest is 9.5, but don't add that one either as it will change anyway; and since we wont have xsi:noNamespaceSchemaLocation you can also delete xmlns:xsi